### PR TITLE
feat: export prometheus metrics

### PR DESCRIPTION
**Pull request by domcyrus**
Export prometheus metrics about storage usage. Implements issue #8

### CODE REVIEW

File: internal/storage/main.go [hunk 1]
```diff
@@ -40,6 +44,19 @@ type ActionEvent struct {
...
+func (ae *ActionEvent) ExportMetrics(promStorage *prometheus.GaugeVec) {
+	for _, claim := range ae.Data.Claims {
+		promStorage.With(prometheus.Labels{
+			"type":        ae.Type,
+			"eventtype":   ae.EventType,
+			"claimenv":    strconv.Itoa(claim.Environment),
+			"claimpvc":    claim.PersisteStorageClaim,
+			"project":     ae.Meta.Project,
+			"environment": ae.Meta.Environment,
+		}).Set(float64(claim.BytesUsed))
```
Comment (shreddedbacon) [inline internal/storage/main.go:56]: I'm not sure how relevant exporting the `type` and `eventType` are, as this is used only by Lagoon to determine how to handle the message. I don't see any reason to capture either of these in the metrics

File: main.go [hunk 2]
```diff
@@ -39,12 +40,20 @@ import (
...
-	scheme   = runtime.NewScheme()
-	setupLog = ctrl.Log.WithName("setup")
+	scheme       = runtime.NewScheme()
+	setupLog     = ctrl.Log.WithName("setup")
+	prom_storage = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "lagoon_storage_calculator_bytes",
+			Help: "The lagoon storage calculator bytes",
+		},
+		[]string{"type", "eventtype", "claimenv", "claimpvc", "project", "environment"},
```
Comment (shreddedbacon) [inline main.go:55]: as above re `type` and `eventtype`

**Review by shreddedbacon — CHANGES_REQUESTED** [review]

**Comment by domcyrus** [comment]
@shreddedbacon I've removed `type` and `eventtype` labels. I've also updated all the dependencies. Is this ok?

**Review by shreddedbacon — CHANGES_REQUESTED** [review]
You'll also need to update the dockerfile go version used https://github.com/domcyrus/storage-calculator/blob/exportPrometheusMetrics/Dockerfile#L2

**Review by shreddedbacon — APPROVED** [review]

**Pull request merged**